Jenkinsfile

```diff
@@ -3,10 +3,12 @@ library changelog: false, identifier: 'lib@master', retriever: modernSCM([
     remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
 ]) _
 
-void runAPItests(String DOCKER_IMAGE_VERSION, CLIENT_VERSION, OWNER) {
+void runAPItests(String DOCKER_IMAGE_VERSION, BRANCH_NAME, API_COMMIT_SHA, CLIENT_VERSION, OWNER) {
     stagingJob = build job: 'pmm2-api-tests', parameters: [
         string(name: 'DOCKER_VERSION', value: DOCKER_IMAGE_VERSION),
+        string(name: 'GIT_BRANCH', value: BRANCH_NAME),
         string(name: 'OWNER', value: OWNER),
+        string(name: 'GIT_COMMIT_HASH', value: API_COMMIT_SHA)
```

Comment:
I always wondered why we use so different strings for names and values. Isn't it confusing? Can we use `GIT_COMMIT_HASH` for both, for example?